src/services/user.service.ts

```diff
@@ -0,0 +1,62 @@
+import { User } from 'src/db/users';
+import UserModel from '../models/user.model';
+import sequelize from 'sequelize';
+
+async function createUser (login: string, password: string, age: number) {
+    await UserModel.create({login,password,age}, {
+        fields:['login','password','age','isDeleted']
+    });
```

Comment:
```suggestion
    await UserModel.create({login,password,age});
```

Don't think we need to use `fields`here